blockchain/standalone/pow_test.go

```diff
@@ -393,3 +393,1676 @@ func TestCheckProofOfWork(t *testing.T) {
+			{176, 518780, 0x1c04770c},
+			{177, 521618, 0x1c04a696},
+			{178, 527138, 0x1c050e88},
+			{179, 527498, 0x1c050fc6},
+			{180, 529412, 0x1c0531b7},
+			{181, 531504, 0x1c055899},
+			{182, 537058, 0x1c05d0f9},
+			{183, 537832, 0x1c05dc56},
+			{184, 552759, 0x1c0768e8},
+			{185, 552972, 0x1c076642},
+			{186, 553461, 0x1c076c05},
+			{187, 554871, 0x1c078e2d},
+			{188, 557624, 0x1c07dbe2},
+			{189, 561595, 0x1c085614},
+			{190, 565227, 0x1c08cb66},
+			{191, 574454, 0x1c0a2606},
+			{192, 576742, 0x1c0a7a33},
+			{193, 581677, 0x1c0b4981},
+			{194, 584505, 0x1c0bc102},
+			{195, 584521, 0x1c0bb357},
+			{196, 585225, 0x1c0bc6d5},
+			{197, 585838, 0x1c0bd5ff},
+			{198, 591383, 0x1c0cdfa8},
+			{199, 596326, 0x1c0dde5c},
+			{200, 598690, 0x1c0e55e7},
+			{201, 601843, 0x1c0f01d2},
+			{202, 606103, 0x1c0ffe18},
+			{203, 606910, 0x1c101f95},
+			{204, 608877, 0x1c108f90},
+			{205, 610610, 0x1c10f252},
+			{206, 613446, 0x1c11a6a6},
+			{207, 613767, 0x1c11a830},
+			{208, 616648, 0x1c126765},
+			{209, 618417, 0x1c12d7d2},
+			{210, 623141, 0x1c143a55},
+			{211, 625762, 0x1c14fec2},
+			{212, 625880, 0x1c14ef11},
+			{213, 625941, 0x1c14da85},
+			{214, 627461, 0x1c15443c},
+			{215, 627469, 0x1c152ac2},
+			{216, 630165, 0x1c15ff66},
+			{217, 637468, 0x1c189ca0},
+			{218, 639356, 0x1c193ef5},
+			{219, 641519, 0x1c1a02db},
+			{220, 641961, 0x1c1a1210},
+			{221, 643700, 0x1c1aadd7},
+			{222, 646353, 0x1c1bb463},
+			{223, 647934, 0x1c1c47c2},
+			{224, 649278, 0x1c1cc1f9},
+			{225, 652899, 0x1c1e5503},
+			{226, 668981, 0x1c271457},
+			{227, 670303, 0x1c27b995},
+			{228, 670467, 0x1c27a36c},
+			{229, 671633, 0x1c283135},
+			{230, 676603, 0x1c2b5231},
+			{231, 680345, 0x1c2dc7f3},
+			{232, 685034, 0x1c311e3c},
+			{233, 692431, 0x1c37092d},
+			{234, 694103, 0x1c3842ad},
+			{235, 694962, 0x1c38c46a},
+			{236, 699094, 0x1c3c5e51},
+			{237, 705075, 0x1c422298},
+			{238, 709010, 0x1c461c6d},
+			{239, 710982, 0x1c4804cc},
+			{240, 713828, 0x1c4b05d6},
+			{241, 716627, 0x1c4e175f},
+			{242, 719236, 0x1c5108e0},
+			{243, 719557, 0x1c510fd5},
+			{244, 721094, 0x1c52af16},
+			{245, 724674, 0x1c572909},
+			{246, 731160, 0x1c604034},
+			{247, 731204, 0x1c5fdb43},
+			{248, 737107, 0x1c68dcc0},
+			{249, 737314, 0x1c68b4a2},
+			{250, 737578, 0x1c68a56d},
+			{251, 738726, 0x1c6a13fd},
+			{252, 745471, 0x1c75a1b8},
+			{253, 748735, 0x1c7b5d43},
+			{254, 749877, 0x1c7d0ae9},
+			{255, 750488, 0x1c7dab62},
+			{256, 754736, 0x1d0085e5},
+			{257, 755780, 0x1d008781},
+			{258, 757308, 0x1d008a34},
+			{259, 761267, 0x1d009291},
+			{260, 761700, 0x1d0092e1},
+			{261, 768268, 0x1d00a269},
+			{262, 771486, 0x1d00aa33},
+			{263, 771983, 0x1d00aabd},
+			{264, 772304, 0x1d00aacc},
+			{265, 775659, 0x1d00b362},
+			{266, 776129, 0x1d00b3e0},
+			{267, 778615, 0x1d00ba4b},
+			{268, 782911, 0x1d00c69d},
+			{269, 785122, 0x1d00ccca},
+			{270, 792634, 0x1d00e5e6},
+			{271, 806060, 0x1d00ffff},
```

Comment:
This was for a 90% decrease.  That said, looking at this again based on your comment, I'm going to update this one to start at a higher initial difficulty in order to avoid clipping at the proof of work limit so quickly which should provide test data that is more likely to discover issues in other implementations.